project/src/components/private-route/private-route.tsx

```diff
@@ -0,0 +1,17 @@
+import {Navigate} from 'react-router-dom';
+import {AppRoute, AuthorizationStatus} from '../../const';
+
+type PrivateRouteProps = {
+  authorizationStatus: AuthorizationStatus;
+  children: JSX.Element;
+}
+
+export default function PrivateRoute(props: PrivateRouteProps): JSX.Element {
```

Comment:
Сразу лучше достать пропсы, используй подход этот в дальнейшем
```suggestion
export default function PrivateRoute({authorizationStatus, children}: PrivateRouteProps): JSX.Element {
```